.circleci/config.yml

```diff
@@ -86,15 +86,11 @@ jobs:
       - restore_cache:
           keys:
             - v1-assets-{{ .Environment.CIRCLE_SHA1 }}
+      - checkout
       - run:
-          name: Setup Heroku
-          command: bash .circleci/setup-heroku.sh
-      - run:
+          name: Deploy Master to Heroku
           command: |
-            git push heroku fan-in-fan-out:master
-            heroku run rake db:migrate
-            sleep 5 # sleep for 5 seconds to wait for dynos
-            heroku restart
+            git push https://heroku:$HEROKU_API_KEY@git.heroku.com/$HEROKU_APP.git master
```

Comment:
This does not work for me, I need to replace the `heroku` username with the user login tied to the used `HEROKU_API_KEY` (`$HEROKU_LOGIN` before this change). Is that expected to work?

Reply:
Hey @m90! Thanks for reporting this: I've moved this conversation over to [an issue](https://github.com/circleci/circleci-docs/issues/2485) if you'd like to join me over there!